themes/nord-color-theme.json

```
@@ -54,7 +53,7 @@
     "editorBracketMatch.background": "#2e344000",
     "editorBracketMatch.border": "#88c0d0",
     "editorCodeLens.foreground": "#4c566a",
-    "editorGroup.background": "#2e3440",
```

The addition of the new `editorGroup.emptyBackground` looks good to me, but again the deprecated key should stay.
Even though it was removed in Microsoft/vscode@a970588eb6ecb6772cd3e09c51a62d72914fdc7d ([VS Code 1.25.0: „Deprecated Theme colors“][rln]), they made sure to keep backwards compatibility Microsoft/vscode#50773.
Please add the deprecated key again including a comment to ensure we know why this key is included:

```suggestion
    /*
     * This key is deprecated as of VS Code version 1.25.0 in favour of `editorGroup.emptyBackground`.
     * See https://github.com/microsoft/vscode/commit/a970588eb6ecb6772cd3e09c51a62d72914fdc7d,
     * https://code.visualstudio.com/updates/v1_25#_deprecated-theme-colors and
     * https://github.com/microsoft/vscode/issues/50773 for more details.
     */
    "editorGroup.background": "#2e3440",
```

[rln]: https://code.visualstudio.com/updates/v1_25#_deprecated-theme-colors